src/authorizations/authorizations.service.ts

```diff
@@ -55,14 +67,33 @@ export class AuthorizationsService {
     }
   }
 
+  private async getAuthRequest(address: string, id: BigNumber): Promise<AuthorizationRequest> {
+    const account = await this.accountsService.findOne(address);
+
+    const identity = await account.getIdentity();
+
+    let authRequest: AuthorizationRequest;
+    if (identity) {
```

Comment:
I am reading this like:

> If an Account has an Identity, then we get the identity auths, otherwise get the account auths.

Will this work? It seems like authorization can target an Identity or an Account.

<img width="594" alt="Screen Shot 2022-10-03 at 2 37 59 PM" src="https://user-images.githubusercontent.com/86971143/193653519-bee631cc-7803-466e-a076-9b3d2393dd33.png">

What about the case where an Authorization targets an Account that already has an Identity? Will falling back to the Identity lookup still work? I am not sure if the assumption this code is making will always work, although it might.

Reply:
I am thinking when there is an identity to call `this.findOne` two times and take either result.

It would be messier and I'd prefer if what you now is sufficient though.

Reply:
I tried some of the authorizations and my logic seems to work. I too am not that happy with the logic here but couldn't figure out anything else. One other way that pops up in my mind is that we add a param specifying that the authorization being accepted is for identity/account, that way instead of guessing based on account identity we would proceed based on user input and if authorization is a mismatch anyhow that would throw an error. What do you think of this @VictorVicente ?

Reply:
I don't think we should ask users for input when we can double query to figure it out.

I think this will work for most cases and is better than the current implementation, but there might be a bug where an auth gets targeted to an Account but we only check the Identity. 

I can check to see if the storage will be transitive thorough the Account for the Identity

Reply:
I was able to reproduce the case I am thinking of. Join an Identity, and make an Auth targeting the account, this will never be able to accept it, even though its a valid auth.

Something like this seems to cover that case:
```ts
   const ignoreNotFound = (error: Error): undefined => {
      if (error instanceof NotFoundException) {
        return undefined;
      } else {
        throw error;
      }
    };

    let authRequest: AuthorizationRequest | undefined;
    if (identity) {
      authRequest = await this.findOne(identity, id).catch(ignoreNotFound);
    }

    if (!authRequest) {
      authRequest = await this.findOne(account, id);
    }

    // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
    return authRequest!;
```

The exception is a bit annoying to deal with, espcially since its already a transformed error. Not sure on the cleanest way to handle it, but I think shoving it into a util function for catch isn't too bad.